qemu/tests/hpt_max_page_size.py

```diff
@@ -38,3 +39,17 @@ def _check_meminfo(key):
     if _check_meminfo("Hugepagesize") != params["expected_value"]:
         test.fail("The hugepage size doesn't match, "
                   "please check meminfo: %s " % _check_meminfo("Huge"))
+    # Please set 1G huge page as default huge page size on power9
+    if params.get("sub_type") == "hugepage_reset":
+        origin_nr = params.get("origin_nr")
+        error_context.context("Setup hugepage number to %s in guest" % origin_nr, logging.info)
+        send_cmd = params.get("send_cmd")
+        if session.cmd_status(send_cmd, timeout):
+            test.fail("Failed to assign nr in the guest")
+        send_cmd_check_result = params.get("send_cmd_check_result")
+        output = session.cmd_output(send_cmd_check_result, timeout)
```

Comment:
Remove the parameter `timeout`, it should only be used to log in guest, not to execute a command. Also L47.

Reply:
Updated